cve_bin_tool/cli.py

```diff
@@ -406,6 +431,9 @@ def main(argv=None, outfile=sys.stdout):
     # update db if needed
     if args.update != "never":
         cvedb_orig.get_cvelist_if_stale()
+    if args.csv:
+        global output_csv_file
```

Comment:
Probably better to let people specify their own filename, and print to console if they don't.